src/plone/api/content.py

```diff
@@ -549,12 +549,11 @@ def find(context=None, depth=None, **kwargs):
     # Convert interfaces to their identifiers
     object_provides = query.get('object_provides', [])
     if object_provides:
-        if not isinstance(object_provides, list):
+        if not isinstance(object_provides, (list, tuple)):
```

Comment:
Why not make the function work with all iterable types? This would work:

```
import collections
if isinstance(object_provides, collections.Iterable):
```